apps/gallery/js/ImageEditor.js

```diff
@@ -1221,8 +1229,51 @@ ImageEditor.prototype.getFullSizeBlob = function(type, done, progress) {
       setTimeout(function() {
         done(blob);
       });
+    }
+
+    function updateExif(blob, metaData, callback) {
+      metaData.Orientation = 1;
+      if (metaData.PixelXDimension) {
+        metaData.PixelXDimension = canvas.width;
+      }
+      if (metaData.PixelYDimension) {
+        metaData.PixelYDimension = canvas.height;
+      }
```

Comment:
I could change it here, but then I need to have code to write date time in exif. I'll file a followup bug instead